### PR TITLE
Change the naming of the phoshpate oxygens to OP1/OP2

### PR DESCRIPTION
**Pull request by amjjbonvin**
You are about to submit a new Pull Request. Before continuing make sure you read the [contributing guidelines](CONTRIBUTING.md) and that you comply with the following criteria:

- [ ] You have sticked to Python. Please talk to us before adding other programming languages to HADDOCK3
- [X] Your PR is about CNS
- [ ] Your code is well documented: proper docstrings and explanatory comments for those tricky parts
- [ ] You structured the code into small functions as much as possible. You can use classes if there is a (state) purpose
- [ ] Your code follows our coding style
- [ ] You wrote tests for the new code
- [ ] `tox` tests pass. *Run `tox` command inside the repository folder*
- [X] `-test.cfg` examples execute without errors. *Inside `examples/` run `python run_tests.py -b`*
- [X] PR does not add any dependencies, unless permission granted by the HADDOCK team
- [X] PR does not break licensing
- [ ] Your PR is about writing documentation for already existing code :fire:
- [ ] Your PR is about writing tests for already existing code :godmode:

---

Modified the nucleic acids topo/param files to change the naming of the phoshpate oxygens to OP1/OP2, consistent with the PDB official nomenclature

### CODE REVIEW

File: examples/docking-protein-DNA/data/cro_air.tbl
Comment (VGPReys) [inline examples/docking-protein-DNA/data/cro_air.tbl:1]: Already here it was known that OP1 and OP2 existed !
But not in the force field.
The Haddock2.X servers have their own machinery to convert from OP1 to O1P ?

File: examples/docking-protein-DNA/data/cro_air.tbl [hunk 1]
```diff
@@ -38,12 +38,12 @@ assign ( resid 33  and segid A)
 !
 ! HADDOCK AIR restraints for 2nd partner
 !
-assign ( resid 3  and segid B and (name O1P or name O2P or name OP1 or name OP2))
+assign ( resid 3  and segid B and (name OP1 or name OP2 or name OP1 or name OP2))
```
Comment (VGPReys) [inline examples/docking-protein-DNA/data/cro_air.tbl:41]:
```suggestion
assign ( resid 3  and segid B and (name OP1 or name OP2))
```

File: examples/docking-protein-DNA/data/cro_air.tbl [hunk 2]
```diff
@@ -38,12 +38,12 @@ assign ( resid 33  and segid A)
...
        (
         ( resid 10  and segid A)
        )  2.0 2.0 0.0
 !
-assign ( resid 4  and segid B and (name O1P or name O2P or name OP1 or name OP2))
+assign ( resid 4  and segid B and (name OP1 or name OP2 or name OP1 or name OP2))
```
Comment (VGPReys) [inline examples/docking-protein-DNA/data/cro_air.tbl:46]:
```suggestion
assign ( resid 4  and segid B and (name OP1 or name OP2))
```

File: examples/docking-protein-DNA/data/cro_air.tbl [hunk 3]
```diff
@@ -72,7 +72,7 @@ assign ( resid 30  and segid B and (name O1P or name O2P or name OP1 or name OP2
         ( resid 42  and segid A)
        )  2.0 2.0 0.0
 !
-assign ( resid 31  and segid B and (name O1P or name O2P or name OP1 or name OP2))
+assign ( resid 31  and segid B and (name OP1 or name OP2 or name OP1 or name OP2))
```
Comment (VGPReys) [inline examples/docking-protein-DNA/data/cro_air.tbl:75]:
```suggestion
assign ( resid 31  and segid B and (name OP1 or name OP2))
```

File: examples/docking-protein-DNA/data/cro_air.tbl
Comment (amjjbonvin) [reply examples/docking-protein-DNA/data/cro_air.tbl:1]: No conversion, it simply regenerate the missing atoms.

**Review by VGPReys — APPROVED** [review]

**Pull request merged**